pyproject.toml

```
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "mcstatus"
+version = "6.5.0"
+description = "A library to query Minecraft Servers for their status and capabilities."
+authors = ["Nathan Adams <dinnerbone@dinnerbone.com>"]
+license = "Apache-2.0"
+readme = "README.md"
+homepage = "https://dinnerbone.com/minecraft/tools/status/"
```

This page is broken. Please link to https://pypi.python.org/pypi/mcstatus instead.

```suggestion
homepage = "https://pypi.python.org/pypi/mcstatus"
```

---

This would already appear on PyPI listing though, pointing homepage back to PyPI is a bit odd, perhaps it would make more sense to just remove it entirely?

---

You're right it's odd, but it is already on the pypi page https://pypi.org/project/mcstatus/
I'm okay with removing the homepage entirely if pypi allows for that to be optional.
I only noticed because of the tar-diff that I did between this branch's sdist and the current version's sdist.